### PR TITLE
Add Gridlines

### PR DESCRIPTION
**Pull request by claytongentry**

### CODE REVIEW

**Review by vcabansag — COMMENTED** [review]
Merge it. I think as we use the gridlines we can circle back to them.

**Comment by claytongentry** [comment]
Scratched out a little extraneous css there. Merging now.

File: source/stylesheets/site.css.scss [hunk 1]
```diff
@@ -247,6 +247,46 @@ body {
...
+  border: 0;
+  border-top: 2px solid orange;
+}
+
+.grid-col {
+  position: absolute;
+  height: 900px;
+  width: 61px;
+  border-left: 2px solid orange;
+  border-right: 2px solid orange;
```
Comment (vcabansag) [inline source/stylesheets/site.css.scss:268]: @claytongentry A good best practice with your CSS is to order the attributes alphabetically. For example, the border-left and border-right would go higher on the list and width goes at the bottom. Makes it easier to scan when you're deep in the CSS/SCSS.

Comment (claytongentry) [reply source/stylesheets/site.css.scss:268]: Ah, missed this before I merged. Will sneak into the next PR.

**Pull request merged**